v1/k8s/use-of-a-specific-secret.rego

```diff
@@ -0,0 +1,75 @@
+package verify
+
+import future.keywords.in
+
+default allow := false
+
+default violations := []
+
+verify = v {
+	v := {
+		"allow": allow,
+		"violation": {
+			"type": "A rule to verify that certain secrets are not accessible",
+			"details": violations,
+		},
+		"summary": [{
+			"allow": allow,
+			"reason": reason,
+			"violations": count(violations),
+		}],
+	}
+}
+
+allow {
+	count(violations) == 0
+}
+
+reason = v {
+	allow
+	v := "No secret match"
+}
+
+reason = v {
+	not allow
+	v := "There is at least one secret which match"
+}
+
+secrets = s {
+	containers := object.remove(input.evidence.predicate.content, {"metadata"})
+	s := {s}
+}
+
+
+
+#TODO:
+# require secrets in the evidence in order to properly check if secrets are matches
+# require a better understanding of how secret objects are formed.
+
+violations = j {
+	j := {r |
+		containers := object.remove(input.evidence.predicate.content, {"metadata"})
+		container := containers[_]
+		secrets := container.secret
+
+        count(secrets) > 0 # Checks if there are any secrets
+
+		secret := secrets[_]
+
+		matches(secret)
+
+		# Must add the proper attributes to display
+		r := { 
+			"name": secret.name,
+			"id": secret.id,
+			"is_expired": true,
+		} 
+
+
+	}
+}
+
+matches(secret) {
+
+	# Must implement a way to check if a secret is expired based off how the object is formed
```

Comment:
Can you implement this one, please?

Reply:
None of the attestation i received had any secrets, so i wasnt able to implement a way to check the secrets